seumich/models.py

```diff
@@ -379,6 +381,23 @@ class StudentClassSiteAssignment(models.Model, SeumichDataMixin):
     def __str__(self):
         return '%s has assignment %s in %s' % (self.student, self.assignment,
                                                self.class_site)
+    # Function to insert breaks in place of newline characters so HTML will
+    # actually render newlines
+    @property
+    def grader_comment(self):
+
+        # Check if grader comment is empty string and exit if so
+        if self.raw_grader_comment is None:
+            return self.raw_grader_comment
+
+        # Functionally, <br> or <br /> would also work, but using <br><br />
+        # complies with XHTML standards, in case that is a necessity of this
+        # project
+        return re.sub(
+            '/\n/g',
+            '<br><br />',
+            self.raw_grader_comment.rstrip()
+        )
```

Comment:
I think there might be a couple of issues with this solution. First, I believe what we see in the UI is actually the literal string "\n", which would look like `\\n` in Python (with a backslash escaping the newline). I believe this is due to some quirk related to the Oracle database (or how it is populated with data). Regardless, I don't think your regular expression will catch these instances, only true newline characters. To test this, I would use the example record I provided in a separate comment, where there are a couple instances of this sequence.

Second, I'm not sure the HTML breaks will show up in the UI. During my testing, I kept seeing the tags. That may be related to the quotes put around the `grader_comment` [here](https://github.com/tl-its-umich-edu/student_explorer/blob/c12320c71cbd54d04b3695b623c5c4efe739d414/seumich/templates/seumich/student_class_site_detail.html#L235). And/or we need to add a Django filter or tag that will render the HTML.

This will take some experimentation and testing. You may also want to check [this `linebreaks` tag](https://docs.djangoproject.com/en/2.2/ref/templates/builtins/#linebreaks) out to see if it might be part of a possible solution.